src/ilp-network-client.ts

```diff
@@ -0,0 +1,14 @@
+import { GetBalanceRequest } from './generated/web/ilp/get_balance_request_pb'
+import { GetBalanceResponse } from './generated/web/ilp/get_balance_response_pb'
+import { SendPaymentRequest } from './generated/web/ilp/send_payment_request_pb'
+import { SendPaymentResponse } from './generated/web/ilp/send_payment_response_pb'
+
+export interface IlpNetworkClient {
```

Comment:
Yeah, I might punt and just leave that one to you since it's unclear to me how much time I'm supposed to spend on this. I'm probably going to try to wrap things up today if I can assuming there are no glaring holes.

I will doc stuff though.